packages/php-wasm/node/src/test/php-request-handler.spec.ts

```diff
@@ -765,3 +765,84 @@ describe('PHPRequestHandler – Loopback call', () => {
 		expect(response.text).toEqual('Starting: Ran second.php! Done');
 	});
 });
+
+describe('PHPRequestHandler – Cookie strategy', () => {
+	it('should persist cookies internally with internal-store strategy', async () => {
+		const handler = new PHPRequestHandler({
+			documentRoot: '/',
+			phpFactory: async () =>
+				new PHP(await loadNodeRuntime(RecommendedPHPVersion)),
+			maxPhpInstances: 1,
```

Comment:
Let's copy this test and also run one that explicitly specifies the `cookieStrategy`, just to make sure the explicit behavior is what we expect

Reply:
I addressed this in https://github.com/WordPress/wordpress-playground/pull/1753/commits/364e7de10fca4d942a4c587fad6e5dc2cca9dba8.